docs/features.md

```diff
@@ -300,27 +306,7 @@ User can register additional custom snippets, for more information on how to con
 
 ### Code actions
 
-| Name                               | Example                                                              |
-|------------------------------------|----------------------------------------------------------------------|
-| Clean namespace require/imports    | <img src="../images/features/clean-ns.gif" width=360>                |
-| Add require                        | <img src="../images/features/add-missing-require.gif" width=360>     |
-| Add known common require           | <img src="../images/features/add-common-require.gif" width=360>      |
-| Add known common import            | <img src="../images/features/add-common-import.gif" width=360>       |
-| Add suggested alias require        | <img src="../images/features/add-alias-suggestion.gif" width=360>    |
-| Create private function            | <img src="../images/features/create-private-function.gif" width=360> |
-| Resolve macro as...                | <img src="../images/features/resolve-macro-as.gif" width=720>        |
-| Inline symbol                      | <img src="../images/features/inline-symbol.gif" width=360>           |
-| Change coll to map,vector,set,list | <img src="../images/features/change-coll.gif" width=720>             |
-| Move coll entry up/down            | <img src="../images/features/move-coll-entry.gif" width=720>         |
-| Move to let                        | <img src="../images/features/move-to-let.gif" width=720>             |
-| Cycle privacy                      |                                                                      |
-| Cycle fn literal                   | <img src="../images/features/cycle-fn-literal.gif" width=480>        |
-| Extract function                   | <img src="../images/features/extract-function.gif" width=360>        |
-| Thread first/all last              | <img src="../images/features/thread-first-all.gif" width=360>        |
-| Unwind thread                      |                                                                      |
-| Sort map                           | <img src="../images/features/sort-map.gif" width=720>                |
-| Suppress diagnostic                |                                                                      |
-| Create test for function           |                                                                      |
+See [below](#clojuser-lsp-extra-commands) for screenshots.
```

Comment:
IMO we should mention both commands and code actions but give more highlights for code actions, as they are smarter than commands, as they are available when they make sense be available.

Reply:
I agree. Or rather, I'd say that @snoe, the wording of the new section strongly emphasizes binding commands, before offhandedly mentioning code actions. I think it'd be a barrier to entry if everyone thought they had to set up their own keybindings to use clojure-lsp. I know you use keybindings more, and that's sort of the vim way. And I like that way too, because I use Emacs in evil mode. But I'd guess that the majority of users are going to favor the code actions. And what @ericdallo has been saying is very true... the code actions guide everyone, but particularly new users, to refactorings that are appropriate for the current cursor position. So, I'd suggest explaining code actions before invoking the "we **strongly** suggest" language for setting up keybindings.

Reply:
Here's the thing, code actions don't really need documentation from us in the way that executeCommand does. It falls in the same bucket as go to def and rename; lsp client features.

Fair enough point about the ordering, so I moved the note about code actions above the keybindings paragraph.

Reply:
I see, even so, as a new clojure-lsp user, I would like to know what features clojure-lsp offer, and mentioning only commands may leave users think this would work only manually.
Well, I'm ok leaving the note as the first thing though